src/migrations/versions/2024_06_17_1521-4265426f8438_create_stes_table.py

```diff
@@ -0,0 +1,44 @@
+"""create stes table
+
+Revision ID: 4265426f8438
+Revises: f84a0ed81bdc
+Create Date: 2024-06-17 15:21:58.003045
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "4265426f8438"
+down_revision: Union[str, None] = "f84a0ed81bdc"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "organization",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("name"),
+    )
+
+    # Add the 'organization_id' column to the 'camera' and 'user' tables and create  foreign key constraints
+    op.add_column("camera", sa.Column("organization_id", sa.Integer(), nullable=True))
```

Comment:
Open question: for the migration to be reversible, how do we fill the org_id value for existing entries?

Reply:
good question : do we really want the migration to be reversible ? :)